doc/source/whatsnew/v3.0.0.rst

```diff
@@ -77,6 +77,7 @@ Other enhancements
 - :meth:`pandas.concat` will raise a ``ValueError`` when ``ignore_index=True`` and ``keys`` is not ``None`` (:issue:`59274`)
 - :py:class:`frozenset` elements in pandas objects are now natively printed (:issue:`60690`)
 - Add ``"delete_rows"`` option to ``if_exists`` argument in :meth:`DataFrame.to_sql` deleting all records of the table before inserting data (:issue:`37210`).
+- Added ``tzdata`` to the ``_hard_dependencies`` (:issue:`61273`)
```

Comment:
`_hard dependencies`  is the internal name of the variable. I think `tzdata` is already required according to our docs, I guess we may want to have a note for a bug where tzdata being installed wasn't checked? @rhshadrach @mroeschke

Reply:
Agreed.  As only user facing-changes go in the whatsnew, I'd be fine with no note here at all. tzdata was being checked and would certainly raise an import error upon import of pandas, adding it to `_hard_dependencies` makes the message a bit more clear. If we do go with a note, I'd suggest it stay in other enhancements but be something like

    Improved error message when importing pandas and ``tzdata`` is not installed (:issue:`61273`)

Reply:
@datapythonista  @rhshadrach Thanks for your suggestions. I have removed the description about the `tzdata` dependency from the whatsnew (1955b05).